modules/pdf_generator/report/procedures.php

```diff
@@ -47,35 +56,35 @@ function procedure_order_data($orderid){
     "WHERE po.procedure_order_id = ?",
     $orderid);
 
-    $content_pro .= "(" . oeFormatShortDate(substr($orow['date'], 0, 10)) . ")<br><table border=\"1\" style=\"background-color: silver\"><tr><td>Patient ID</td>".
+    $content_pro .= "(" . oeFormatShortDate(substr($orow['date'], 0, 10)) . ")<br><table border=\"1\" style=\"background-color: silver\"><tr><td>" . xlt("Patient ID") . "</td>".
                     "<td>" . $orow['pid'] . "</td>".
-                    "<td>Order ID</td>".
+                    "<td>" . xlt("Order ID") . "</td>".
                     "<td>" . $orow['procedure_order_id'] ;
-    
+
     if ($orow['control_id']) {
-        $content_pro .= " Lab: " . $orow['control_id'];
+        $content_pro .= xlt(" Lab: ") . $orow['control_id'];
     }
     $content_pro .= "</td></tr>";
 
-    $content_pro .= "<tr><td>Patient Name</td>".
+    $content_pro .= "<tr><td>" . xlt("Patient Name") . "</td>".
                     "<td>" . $orow['lname'] . ', ' . $orow['fname'] . ' ' . $orow['mname'] . "</td>".
-                    "<td>Ordered By</td>".
+                    "<td>" . xlt("Ordered By") . "</td>".
                     "<td>" . $orow['ulname'] . ', ' . $orow['ufname'] . ' ' . $orow['umname'] . "</td></tr>";
 
-    $content_pro .= "<tr><td>Order Date</td>".
+    $content_pro .= "<tr><td>" . xlt("Order Date") . "</td>".
                     "<td>" . $orow['date_ordered'] . "</td>".
-                    "<td>Print Date</td>".
+                    "<td>" . xlt("Print Date") . "</td>".
                     "<td>" . oeFormatShortDate(date('Y-m-d')) ."</td></tr>";
-    
-    $content_pro .= "<tr><td>Order Status</td>".
+
+    $content_pro .= "<tr><td>" . xlt("Order Status") . "</td>".
                     "<td>" . $orow['order_status'] . "</td>".
-                    "<td>Encounter Date</td>".
+                    "<td>" . xlt("Encounter Date") . "</td>".
                     "<td>" . oeFormatShortDate(substr($orow['date'], 0, 10)) . "</td></tr>";
 
-    $content_pro .= "<tr><td>Lab</td>".
+    $content_pro .= "<tr><td>" . xlt("Lab") . "</td>".
                     "<td>" . $orow['labname'] . "</td>";
-    if($orow['specimen_type'])  $content_pro .= "<td>Specimen Type</td>";
-                    
+    if($orow['specimen_type'])  $content_pro .= "<td>" . xlt("Specimen Type") . "</td>";
+
     $content_pro .= "<td>" . $orow['specimen_type'] . "</td></tr></table><br><br>";
```

Comment:
Good that you included the xlt functions but these green lines and red stuffs makes my eyes go :roll_eyes:  will test and get back to a line id needed